manager/src/main/java/com/redhat/service/smartevents/manager/api/user/SchemaAPI.java

```diff
@@ -93,6 +95,46 @@ public Response getCatalog() {
         return Response.ok(response).build();
     }
 
+    @APIResponses(value = {
+            // we can't use JsonSchema.class because of https://github.com/swagger-api/swagger-ui/issues/8046
+            @APIResponse(description = "Success.", responseCode = "200",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = JsonNode.class))),
+            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON)),
+            @APIResponse(description = "Unauthorized.", responseCode = "401"),
+            @APIResponse(description = "Forbidden.", responseCode = "403"),
+            @APIResponse(description = "Internal error.", responseCode = "500", content = @Content(mediaType = MediaType.APPLICATION_JSON))
+    })
+    @Operation(summary = "Get source processor schema", description = "Get the source processor JSON schema.")
+    @GET
+    @Path("/sources/{name}")
+    public Response getSourceProcessorSchema(@PathParam("name") String name) {
```

Comment:
There may be an UX issue.
The `/api/v1/schemas` endpoint returns a list of supported actions/sources where name is a name itself without `.json` suffix:
```
    {
      "kind": "ProcessorSchemaEntry",
      "name": "Slack",
      "type": "action",
      "href": "/api/v1/schemas/actions/Slack.json"
    }
```
However here the name is expected to be whole file name with the `.json` suffix, that is quite confusing.

Reply:
@sutaakar yeah. Shall we remove the suffix `.json` at all?

Reply:
I've removed it, let me know wdyt @sutaakar @danielezonca

Reply:
Fine for me to remove the suffix